apps/system-e2e/src/tests/islandis/service-portal/smoke/financial-payment-application.spec.ts

```diff
@@ -49,30 +49,40 @@ test.describe('MS - Fjármál overview', () => {
         `role=button[name="${label(m.scheduleApplication)}"]`,
       )
 
-      const pagePromise = context.waitForEvent('page')
-      applicationButton.click()
-      const newPage = await pagePromise
-      await newPage.waitForLoadState()
+      const [delegationPopup] = await Promise.all([
+        page.waitForEvent('popup'),
+        // Click - Open Link in Popup
+        applicationButton.click(),
+      ])
 
+      // Choose delegation
+      await delegationPopup
+        .getByRole('button', { name: 'Gervimaður Færeyjar' })
+        .click()
       // Assert
-      expect(newPage.url()).toContain('umsoknir/greidsluaaetlun')
+      await expect(
+        delegationPopup.getByText('Viltu nota gervigögn?'),
```

Comment:
Should this use the translation string?

Reply:
I was not sure which text to use since there were 2 🤷🏼‍♂️. Suggestions?

Reply:
It's hardcoded in the application since this is not for production. So this is perfectly fine.